### ISSUE TITLE
wrong camera angle when switch to Cross Section View after zooming in on Plan View

### ISSUE THREAD

**Issue by Algorush** [3DStreet/3dstreet#693]
I found another issue with ortho camera. If zoom in on the camera in Plan view and then switch to Cross Section View, the camera angle takes on random values, and this is not related to changes in window resize:
![after_zoom](https://github.com/3DStreet/3dstreet/assets/32197767/c24ccb6e-c0a6-4d06-85d8-dba3fd884ef6)

**Comment by vincentfretin** [comment]
Even after #694 changes, I indeed had that issue on 3dstreet, and also on my project when switching to ortho face view, but I had the issue one time, it's really random, I can't reproduce the issue consistently.

**Comment by vincentfretin** [comment]
I reproduced this weird view if I focus an object in the scene graph by double clicking the item. I'll disable focus when in orthographic camera.
